### PR TITLE
Changes to bgpPeerP_service.go  and bgp_peer_p.go

### PR DESCRIPTION
**Pull request by shrsr**

### CODE REVIEW

File: client/bgpPeerP_service.go [hunk 1]
```diff
@@ -98,3 +99,83 @@ func (sm *ServiceManager) ReadRelationbgpRsPeerPfxPolFromBgpPeerConnectivityProf
...
+
+	var output map[string]interface{}
+	err_output := json.Unmarshal([]byte(containerJSON), &output)
+
+	if err_output != nil {
+		return err_output
+	}
+	for _, value := range output {
+		if rec, ok := value.(map[string]interface{}); ok {
+			for _, val2 := range rec {
```
Comment (lhercot) [inline client/bgpPeerP_service.go:128]: what is val2? Can you put a better name for it?

File: client/bgpPeerP_service.go [hunk 2]
```diff
@@ -98,3 +99,83 @@ func (sm *ServiceManager) ReadRelationbgpRsPeerPfxPolFromBgpPeerConnectivityProf
...
+				if rec2, ok := val2.(map[string]interface{}); ok {
```
Comment (lhercot) [inline client/bgpPeerP_service.go:129]: what is rec2? Can you put a better name for it?

File: client/bgpPeerP_service.go [hunk 3]
```diff
@@ -98,3 +99,83 @@ func (sm *ServiceManager) ReadRelationbgpRsPeerPfxPolFromBgpPeerConnectivityProf
...
+	}
+
+	var output map[string]interface{}
+	err_output := json.Unmarshal([]byte(containerJSON), &output)
+
+	if err_output != nil {
+		return err_output
+	}
+	for _, value := range output {
+		if rec, ok := value.(map[string]interface{}); ok {
```
Comment (lhercot) [inline client/bgpPeerP_service.go:127]: what is rec? Can you put a better name for it?

Comment (shrsr) [reply client/bgpPeerP_service.go:127]: rec is a map of all the values of bgpRsPeerToProfile
It is not required here, so removing it.
Will use a better name if used in other resources.

**Pull request merged**